gradle/sonar.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'org.sonarqube'
+
+// disable sonar for pull request builds until
+// sonar plugin for pull request comments is available at sonarcloud.io
+if (!new Boolean(System.env.TRAVIS_PULL_REQUEST)) {
```

Comment:
According to [javadoc](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#Boolean-java.lang.String-), `Boolean::new` will return true if and only if given string is `"true"`. But `TRAVIS_PULL_REQUEST` is always `"false"` or `PR number`, so this `!new Boolean(System.env.TRAVIS_PULL_REQUEST)` is always true.

And I think it's better to use `TRAVIS_BRANCH` instead, then we can avoid analysis even for the branch which is pushed to `spotbugs` project directly.